paddle/fluid/operators/detail/test_serde.cc

```diff
@@ -75,10 +77,30 @@ void RunSerdeTestTensor(platform::Place place) {
     EXPECT_FLOAT_EQ(tensor_data[i], 31.9);
   }
 
+  // message binary
+  std::string str;
+  varmsg.SerializeToString(&str);
+
+  // message bytebuffer
+  ::grpc::Slice slices_2[1];
```

Comment:
Could you test with a bigger sized buffer? A 1 sized buffer perhaps is not general enough. Or even better, test in a loop, like `for i := 1; i < 100; i++`.

Reply:
There are two styles of repeated field: packed and not packed.
- packed: <tag,data> <tag,data>....
- not packed: <tag,len><data, data,data..>...
This code only guarantees we can parse manual serialized buffer and the standard buffer.